lasagne/layers/helper.py

```diff
@@ -349,7 +358,8 @@ def get_all_params(layer, **tags):
     True
     """
     layers = get_all_layers(layer)
-    params = chain.from_iterable(l.get_params(**tags) for l in layers)
+    params = chain.from_iterable(l.get_params(
+                     unwrap_shared=unwrap_shared, **tags) for l in layers)
```

Comment:
I meant 8 characters from the beginning of `params` here. For a line break directly after an opening brace, in this codebase we usually do a double indent (= 8 characters instead of 4) in the next line.